src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/string.txt

```diff
@@ -291,4 +291,88 @@ true
 "ab"
 
 >> Concatenate(Text(1/0, "000"), "b", Text(7))
-#Error
\ No newline at end of file
+#Error
+
+// Scalar + Table
+>> Concatenate("a", Table({b:"1"},{b:"2"}))
+[{Result:"a1"},{Result:"a2"}]
+
+>> Concatenate("a", LastN(Table({b:"1"},{b:"2"}), 0))
+[]
+
+>> Concatenate("a", If(1<0, ["txt"]))
+[]
+
+>> Concatenate("a", If(1/0<2, ["txt"]))
+#Error
+
+>> Concatenate("a", ["hello", Mid("great", -1), "world"])
+[{Result:"ahello"},{Result:Microsoft.PowerFx.Core.Public.Values.ErrorValue},{Result:"aworld"}]
+
+>> Concatenate("a", Table({a:"one"},If(1<0,{a:"two"}),{a:"three"}))
+[{Result:"aone"},{Result:"a"},{Result:"athree"}]
+
+>> Concatenate("a", Table({a:"one"},If(1/0<2,{a:"two"}),{a:"three"}))
+Microsoft.PowerFx.Core.Public.Values.ErrorValue
+
+>> Concatenate(Blank(), ["hello", Mid("great", -1), "world"])
+[{Result:"hello"},{Result:Microsoft.PowerFx.Core.Public.Values.ErrorValue},{Result:"world"}]
+
+>> Concatenate(Blank(), Table({a:"one"},If(1<0,{a:"two"}),{a:"three"}))
+[{Result:"one"},{Result:""},{Result:"three"}]
+
+>> Concatenate(Blank(), Table({a:"one"},If(1/0<2,{a:"two"}),{a:"three"}))
+Microsoft.PowerFx.Core.Public.Values.ErrorValue
+
+// Table + Scalar (Reverse Arguments)
+>> Concatenate(Table({b:"1"},{b:"2"}), "a")
+[{Result:"1a"},{Result:"2a"}]
+
+>> Concatenate(LastN(Table({b:"1"},{b:"2"}), 0), "a")
+[]
+
+>> Concatenate(If(1<0, ["txt"]), "a")
+[]
+
+>> Concatenate(If(1/0<2, ["txt"]), "a")
+#Error
+
+>> Concatenate(["hello", Mid("great", -1), "world"], "a")
+[{Result:"helloa"},{Result:Microsoft.PowerFx.Core.Public.Values.ErrorValue},{Result:"worlda"}]
+
+>> Concatenate(Table({a:"one"},If(1<0,{a:"two"}),{a:"three"}), "a")
+[{Result:"onea"},{Result:"a"},{Result:"threea"}]
+
+>> Concatenate(Table({a:"one"},If(1/0<2,{a:"two"}),{a:"three"}), "a")
+Microsoft.PowerFx.Core.Public.Values.ErrorValue
```

Comment:
This is not correct - it should return a table with three elements: "onea", <error>, "threea"

Reply:
This is correct, all functions return an error if any argument is an error.

Reply:
The Table function should not return an error in this case. Not a blocker for this PR, but it should be addressed soon.